ppr-ui/src/components/common/PidNumber.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <v-row id="pid-number" class="pid-text-input" no-gutters>
+    <v-col cols="12">
+      <p class="font-weight-bold">PID Number</p>
+    </v-col>
+    <v-col cols="12" sm="3">
+      <v-text-field
+        id="pid-one-input"
+        ref="pidOneRef"
+        maxlength="3"
+        filled persistent-hint autofocus
+        hint="Parcel identifier must contain 9 digits"
+        :readonly="enablePidLoader"
+        :error-messages="invalidPidMsg"
+        v-model="pidOne"
+      />
+    </v-col>
+
+    <v-divider class="horizontal-divider pb-1" />
+
+    <v-col cols="12" sm="3">
+      <v-text-field
+        id="pid-two-input"
+        ref="pidTwoRef"
+        filled
+        maxlength="3"
+        :readonly="enablePidLoader"
+        v-model="pidTwo"
+      />
+    </v-col>
+
+    <v-divider class="horizontal-divider pb-1" />
+
+    <v-col cols="12" sm="3">
+      <v-text-field
+        id="pid-three-input"
+        ref="pidThreeRef"
+        filled
+        maxlength="3"
+        :readonly="enablePidLoader"
+        v-model="pidThree"
+      />
+    </v-col>
+
+    <v-col cols="12" sm="1">
+      <v-progress-circular
+        v-if="enablePidLoader"
+        indeterminate
+        color="primary"
+      ></v-progress-circular>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+/* eslint-disable no-unused-vars */
+import vue from 'vue'
+import { computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useInputRules } from '@/composables'
+/* eslint-enable no-unused-vars */
+
+export default defineComponent({
+  name: 'PidNumber',
+  components: {},
+  props: {},
+  setup (props, context) {
+    // Composable(s)
+    const {
+      isNumber
+    } = useInputRules()
+
+    const localState = reactive({
+      pidOne: '',
+      pidTwo: '',
+      pidThree: '',
+      enablePidLoader: false,
+      pidNumber: computed(() => {
+        return `${localState.pidOne}-${localState.pidTwo}-${localState.pidThree}`
+      }),
+      isValidPid: computed(() => {
+        return (
+          (localState.pidOne ? /^\d+$/g.test(localState.pidOne) : true) &&
```

Comment:
Do we need to add a rule to min length of 3 digits?

Reply:
Currently there is no validations in the validation matrix, so sounds like this will be applied once those requirements have been figured out. 

The pid as a whole however will most likely be an 11 char min/max to include the 9 digits and 2 dashes.
The validation on this is a bit* weird, we will have to apply the validation error to only the one field but the value would be the result of the 3 fields if that makes sense. 
I've done a little bit of that already with the custom error and hint etc

Reply:
yea ok that makes sense, thanks